tests/unit_test.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash -e
 # This is for use as a Github CI Pull Request Unit Test.
-echo "ALL_CHANGED FILES: ${ALL_CHANGED_FILES}."
-echo "CHANGED_PACKAGES: ${CHANGED_PACKAGES}."
+set -e
+echo "CHANGED_PACKAGES: ${CHANGED_PACKAGES}"
 cd /usr/local/lib/crew/packages/
+yes | crew update
 yes | crew upgrade
-(cd ~ && \
-git clone --depth=1 https://github.com/chromebrew/chromebrew.git build_test && \
-cd build_test && \
-yes | CREW_CACHE_ENABLED=1 crew build -vf packages/hello_world_chromebrew.rb)
+git clone --depth=1 --branch="$CREW_BRANCH" "$CREW_REPO" ~/build_test
+# crew wont let you build if you're in the installation directory.
```

Comment:
Thanks for documenting this!

Reply:
Would it make sense to clone into `/tmp/build_test` to indicate this is a temporary location?  Also, remove when tests are complete?

Reply:
We don't really need to clean up after ourselves given that this is being run in a docker image, which will be destroyed at the end of the CI.